workspace/environment.go

```diff
@@ -0,0 +1,83 @@
+package workspace
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"text/template"
+
+	"github.com/joho/godotenv"
+	"github.com/yourbase/yb/plumbing/log"
+	"github.com/yourbase/yb/runtime"
+)
+
+// checkAndSplitEnvVar returns two empty strings and false
+// if env isn't formed as "something=.*"
+func checkAndSplitEnvVar(env string) (name, value string, sane bool) {
+	s := strings.SplitN(env, "=", 2)
+	if sane = (len(s) == 2); sane {
+		name = s[0]
+		value = s[1]
+	}
+	return
+}
+
+// parseEnvironment sanely checks and process an arbitrary number of
```

Comment:
nit: "sanely" doesn't add anything descriptive to the documentation and furthers stigmatization of mental health issues.

```suggestion
// parseEnvironment checks and process an arbitrary number of
```

Reply:
Oh, sorry. Thanks for pointing this out.